docs/guide_about_files_generated.md

```diff
@@ -0,0 +1,305 @@
+Guide
```

Comment:
Ok @daren-thomas. I will work on this. Need to get a quick guide on how to write .rst files :).